metrics-proxy/pom.xml

```diff
@@ -64,6 +64,13 @@
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <!-- Not directly used in this module, but needed to get Import-Packages for JDK packages it exports. -->
```

Comment:
Is jdisc_core always required as provided dependency for bundles?

Reply:
It exports 'javax.*' packages and so on, so I would in general say 'yes'.

Reply:
OK, so it is not necessary for java.* packages.

Reply:
No, because OSGi (at least in r7) still allows not importing them.